zebrad/src/components/sync/end_of_support.rs

```diff
@@ -0,0 +1,92 @@
+//! End of support checking task.
```

Comment:
We should always document new features in the relevant places and in particular where the behaviour significantly difers from `zcashd`. 
So we should add a Zebra book entry, probably under "[Running Zebra](https://zebra.zfnd.org/user/run.html)" and we should definitely highlight it in the `CHANGELOG` and release notes when we tag the next release.
However, given that we don't have many users yet, I don't think we need a blogpost. I also don't see it as something that needs to go in the `README`, I seem to remember people find it too long and overwhelming as it is and I'm not sure that this is some detail that people would go looking for in there.

Reply:
After looking at where this is mentioned in the `zcashd` book again, I should add that the location in the zebra book (under "Running Zebra") can be temporary. 

From the stable release onwards, we might decide we want to dedicate a whole section in the zebra book to Zebra releases to include info such as release cadence and end of support details for each zebra release version. 

I don't think we're there yet though.

Reply:
ah right, this section seems to be more of a section for zebra developers rather than zebra users though. 

Anyway, if we just want to inform users of this particular feature we should probably put it somewhere in the user documentation section of the book.

Reply:
I think we might want to move some of `dev/release-process.md` into the `user` side of the book before the stable release. But that can happen when we've worked out what we want there.